upstream-community-operators/sematext/sematext-operator.v1.0.8.clusterserviceversion.yaml

```diff
@@ -0,0 +1,167 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: sematext-operator.v1.0.8
+  namespace: placeholder
+  annotations:
+    capabilities: "Basic Install"
+    categories: "Monitoring, Logging & Tracing"
+    createdAt: 2019-06-13T08:00:00Z
+    certified: "false"
+    description: |-
+      Full stack observability with Sematext Monitoring Platform!
+    containerImage: docker.io/sematext/sematext-operator:1.0.8
+    support: Sematext, Inc.
+    repository: https://github.com/sematext/sematext-operator
+    alm-examples: |-
+      [{
+        "apiVersion": "sematext.com/v1alpha1",
+        "kind": "SematextAgent",
+        "metadata": {
+          "name": "basic-agent-deployment"
+        },
+        "spec": {
+          "region": "EU"
+        }
+      }]
+spec:
+  customresourcedefinitions:
+    owned:
+    - kind: SematextAgent
+      displayName: Sematext Agent DaemonSet
+      description: Represents a Sematext Agent running on each node of your cluster.
+      name: sematextagents.sematext.com
+      version: v1alpha1
+  description: |-
+    # Sematext Operator
```

Comment:
Hi @komljen,
1. Please remove this line, it will create double caption. You can test a look here https://operatorhub.io/preview. 
2. I need to to know if you tested operator from functional point of view.

This 2 point are must, I will add another points which can make the operator better:
3. add a "channel" - e.g. stable, alpha
4. Add resources to owned CRDs
Add a spec descriptor for region
Add a status descriptor for conditions
Add a status descriptor for deployedRelease